src/main/java/lesson17/OJDBCUtils.java

```diff
@@ -17,4 +20,38 @@ public static Connection getConnection() {
         }
         return connection;
     }
+
+    public static ResultSet getResultSet(String query) throws SQLException {
+        Connection connection = getConnection();
+        PreparedStatement stmt = connection.prepareStatement(query);
+        return stmt.executeQuery();
+    }
+
+
+    public static Orders newOrderByRs(ResultSet rs, Products product) throws SQLException {
+        return new Orders(rs.getBigDecimal("order_num"), rs.getDate("order_date"),
+                rs.getBigDecimal("CUST"), rs.getBigDecimal("rep"),
+                rs.getString("mfr"),
+                rs.getBigDecimal("qty"), rs.getBigDecimal("amount"), product);
+    }
+
+    public static Products newProductByRs(ResultSet rs) throws SQLException {
+        return new Products(rs.getString("MFR_ID"), rs.getString("PRODUCT_ID"),
+                rs.getString("DESCRIPTION"), rs.getBigDecimal("PRICE"), rs.getBigDecimal("QTY_ON_HAND"));
+    }
+    public static Offices newOfficeByRs(ResultSet rs) throws SQLException {
+        return new Offices(rs.getBigDecimal("office"), rs.getString("city"),
+                rs.getString("region"), rs.getBigDecimal("mgr"),
+                rs.getBigDecimal("target"), rs.getBigDecimal("sales"));
+    }
```

Comment:
It may be better to put these methods into a another class, and not where the connection to the database.

Reply:
It is in Utils class. Why it sholdnt by in the same class with connection?

Reply:
It's my opinian, may be  SergeyPasko tell us about this situation.